spec/dor/workflow_object_spec.rb

```diff
@@ -22,10 +22,8 @@
       @item.workflowDefinition.content = '<workflow-def id="accessionWF"/>'
     end
 
-    it 'indexes the number of archived objects for the workflow' do
-      expect(Dor::Config.workflow.client).to receive(:count_archived_for_workflow).and_return(5)
-      expect(@item.to_solr).to include 'accessionWF_archived_isi' => 5
+    it 'indexes the workflow name' do
+      expect(@item.to_solr).to include 'workflow_name_ssim' => ['accessionWF']
     end
   end
```

Comment:
It seems the spec on the `to_solr` method did not previously check for the name, but it was already created by some other part of the `item.to_solr` machinery.  These diffs suggest that removing the `to_solr` method on the workflow object would cause this spec to fail with a method missing error.